common/tests/conftest.py

```diff
@@ -20,6 +20,7 @@ def set_frozen_time():
 @pytest.fixture(autouse=True)
 def force_english(settings):
     settings.LANGUAGE_CODE = 'en'
+    settings.EVENT_MINIMUM_DAYS_BEFORE_START = 7
```

Comment:
maybe change the name of the fixture since it now does a bit more than forcing english?

Reply:
good point! 👍